### ISSUE TITLE
How to specify an Empty Rollback using SQL Dialect

### ISSUE THREAD

**Issue by awhitford-cip** [liquibase/liquibase#3069]
## Environment

**Liquibase Version**: 4.13
**Liquibase Integration & Version**: GitHub Action
**Liquibase Extension(s) & Version**: 7
**Database Vendor & Version**: Snowflake
**Operating System Type & Version**: Linux
**Infrastructure Type/Provider**: Azure

## Description

Unable to specify an empty rollback for a changeset using the SQL dialect.

## Steps To Reproduce

Using a SQL dialect, consider a change set like:
```sql
-- liquibase formatted sql

-- changeset anthonyr:create-table-AW_TEST-1 runOnChange:true
create or replace table AW_TEST (
  ID int not null autoincrement,
  valueDate date not null
)
-- rollback
```
When I test a rollback, Liquibase throws an error:
```
Unexpected error running Liquibase: No inverse to liquibase.change.core.RawSQLChange created
```

I am legitimately OK with an [empty](https://docs.liquibase.com/change-types/empty.html) rollback.  Note that I don't really see a SQL Example in the documentation.

## Actual Behavior

When I test a rollback, Liquibase throws an error:
```
Unexpected error running Liquibase: No inverse to liquibase.change.core.RawSQLChange created
```

## Expected/Desired Behavior

When I test a rollback, Liquibase should basically do nothing for this changeset since the rollback is empty.

**Comment by awhitford-cip** [comment]
My workaround is to specify something benign like:
```sql
-- rollback select 'N/A'
```
But that doesn't seem to be leveraging the _empty_ concept.

**Comment by nvoxland** [comment]
Since just -- rollback apparently does not work there's a few options we could do:
1. Fix it so an empty rollback block counts as "don't do a rollback"
2. Define a keyword (or multiple) like --rollback empty or -- rollback none

I'd tend to think that #2 is better since it's more explicitly saying "there is no rollback" vs. maybe something going wrong and it unexpectedly being empty and so not doing anything on rollback.  And I'd tend to think just one is good. "None" feels like a better description than "empty". "n/a" could work but seems less obvious for people to try, and less understandable to non-native-English speakers?

So I'd suggest defining `-- rollback none` as the way to specify "no rollback needed". But looking for feedback and thoughts.

**Comment by awhitford-cip** [comment]
I would think that `rollback empty` would be more consistent with the [empty](https://docs.liquibase.com/change-types/empty.html) change type.  (In fact, at present `empty` does not really have a SQL dialect.)

**Comment by rorysaur** [comment]
Apparently the docs are also just wrong?: https://docs.liquibase.com/workflows/liquibase-community/liquibase-auto-rollback.html
(where it has a code sample that uses `--rollback empty`)

I tried that and just got a syntax error because it tried to run "empty" as SQL.

**Comment by filipelautert** [comment]
@rorysaur sory for that, seems the documentation team jumped the gun on this one - this feature will be released in Liquibase 4.18 . Meanwhile you need to use an workaround as `-- rollback select 'N/A'` or something like that .

**Comment by rorysaur** [comment]
Thank you @filipelautert! Yes I should've mentioned that I saw the workaround in this thread and it worked for me 👍🏻 glad to hear it will be released soon.

**Comment by ErfanAhmed** [comment]
> @rorysaur sory for that, seems the documentation team jumped the gun on this one - this feature will be released in Liquibase 4.18 . Meanwhile you need to use an workaround as `-- rollback select 'N/A'` or something like that .

I have just tried Liquibase 4.18 with `-- rollback empty`   - didn't work. However, with `-- rollback select 'N/A'`   liquibase don't give an error. The rollback command runs but it deletes the latest entry/row from ``databasechangelog`  table. These causes error while running `update`  command.
